thirdeye/thirdeye-frontend/tests/acceptance/self-serve-import-metric-test.js

```diff
@@ -0,0 +1,168 @@
+import $ from 'jquery';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { selfServeConst } from 'thirdeye-frontend/tests/utils/constants';
+import { visit, fillIn, click, currentURL, triggerKeyEvent } from '@ember/test-helpers';
+
+module('Acceptance | import metric', function(hooks) {
+  setupApplicationTest(hooks);
+
+  const importLinkText = 'Import a Metric From InGraphs';
+  const returnLinkText = 'Back to Create';
+  const inGraphsLinkText = 'Go to InGraphs';
+  const dashboardToImport = 'thirdeye-all';
+  const importWarning = 'Warning... This dashboard name cannot be found in inGraphs. Please verify.';
+  const importSuccessMsgTitle = 'Success! Metrics On-boarded';
+  const importSuccessMsgText1 = `You have successfully onboarded the ${dashboardToImport} dataset , including the following metrics`;
+
+  test(`visiting alert creation page to navigate to import page`, async (assert) => {
+    await visit(`/self-serve/create-alert`);
+
+    assert.equal(
+      $(selfServeConst.SECONDARY_LINK).get(0).innerText,
+      importLinkText,
+      'Import link appears.'
+    );
+
+    await click(selfServeConst.SECONDARY_LINK);
+
+    assert.equal(
+      currentURL(),
+      '/self-serve/import-metric',
+      'Navigation to import page succeeded');
+    assert.ok(
+      $(`${selfServeConst.SECONDARY_LINK}:contains(${returnLinkText})`).get(0),
+      'Return link appears.'
+    );
+    assert.ok(
+      $(`${selfServeConst.SECONDARY_LINK}:contains(${inGraphsLinkText})`).get(0),
+      'InGraph link appears.'
+    );
+  });
+
+  test(`visiting metric import page to import existing dashboard`, async (assert) => {
+    await visit(`/self-serve/import-metric`);
+
+    const $dashboardInput = $(selfServeConst.IMPORT_DASHBOARD_INPUT);
+    let $submitButton = $(selfServeConst.SUBMIT_BUTTON);
```

Comment:
any reason the submit button is a `let` while dashboardInput is a `const` ?

Reply:
I need to cache the element again, as it has changed.